src/pages/index.js

```diff
@@ -27,47 +27,52 @@ const IndexPage = ({
     >
       <html lang="en" />
     </Helmet>
-    <Grid>
-      <Padding bottom={{ smallPhone: 0, smallTablet: 2, desktop: 2 }}>
-        <CaseStudyPreview
-          caseStudy={content.featuredCaseStudy}
-          subHeading="Featured work"
-        />
-      </Padding>
-      <Padding bottom={{ smallPhone: 0, desktop: 2 }} />
-    </Grid>
-    <GreyBackground topMargin>
+    <main>
```

Comment:
was this main added because of a lighthouse check? If so, shouldn't it be present on our `Layout` instead?

Reply:
Good point! Done